### DISCUSSION TITLE
Welcome to AgentsPy Discussions!

### DISCUSSION THREAD

**Discussion by dybber** [DatalogiForAlle/AgentsPy#132] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JulianLevin** [comment]
Hej.

Jeg underviser i informatik og bruger jeres materiale om 'Agent-baseret simulering i Python'. Det er virkelig godt, men jeg kan ikke få add_toggle_botton til at virke i økosystem med rovdyr og byttedyr? Kan du hjælpe? Det virker ikke med en alm. knap.

mvh. Julian Levin, Mulernes Legatskole, Odense
